blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,75 @@
+:root {
+  --v2-feature-carousel-padding: 32px 0;
+  --v2-feature-carousel-card-padding: 24px;
+  --v2-feature-carousel-card-width: 150px;
+  --v2-feature-carousel-card-height: 200px;
+  --v2-feature-carousel-icon-size: 48px;
+  --v2-feature-carousel-svg-size: 24px;
+  --v2-feature-carousel-border-radius: var(--border-radius);
+}
+
+.v2-feature-carousel-container {
+  background-color: var(--c-grey-50);
+}
+
+.v2-feature-carousel-container .v2-feature-carousel-wrapper {
+  max-width: 1330px;
+  width: 100%;
+}
+
+.v2-feature-carousel {
+  padding: var(--v2-feature-carousel-padding);
+}
+
+.v2-feature-carousel .v2-feature-carousel__cards-list {
+  display: flex;
+  gap: 16px;
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  flex-flow: row nowrap;
+}
+
+.v2-feature-carousel__card {
+  display: flex;
+  flex: 0 0 auto;
+  width: var(--v2-feature-carousel-card-width);
+  height: var(--v2-feature-carousel-card-height);
+  padding: var(--v2-feature-carousel-card-padding);
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  border-radius: var(--v2-feature-carousel-border-radius);
+  background: var(--c-white);
+  scroll-snap-align: center;
+  margin: auto;
```

Comment:
This margin auto needs to be only on the first item of the list (margin-left: auto) and last item of the list (margin-right: auto) because if not we can see something like the image where the spaces btw the elements are not correct.

`.v2-feature-carousel__card:first-child { margin-left: auto; }
.v2-feature-carousel__card:last-child { margin-right: auto; }
`
<img width="1057" alt="image" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/3090999/93cca7de-c493-4608-ac99-f26cfb2483d3">

Reply:
Update, missed that :)